conf/app.routes

```diff
@@ -10,4 +10,6 @@ GET     /api/teams/:team/slug-dependencies                 @uk.gov.hmrc.serviced
 GET     /api/repository-name                               @uk.gov.hmrc.servicedependencies.controller.ServiceDependenciesController.repositoryName(group: String, artefact: String, version: String)
 GET     /api/repositories/:repository/module-dependencies  @uk.gov.hmrc.servicedependencies.controller.ServiceDependenciesController.moduleDependencies(repository: String, version: Option[String])
 GET     /api/latest-version                                @uk.gov.hmrc.servicedependencies.controller.ServiceDependenciesController.latestVersion(group: String, artefact: String)
+GET     /api/meta-artefact/serviceDeps                     @uk.gov.hmrc.servicedependencies.controller.ServiceDependenciesController.metaArtefactDependencies(flag: SlugInfoFlag ?= SlugInfoFlag.Latest, repoType: Option[RepoType], group: Option[String], artefact: Option[String], versionRange: Option[BobbyVersionRange], scope: Option[List[DependencyScope]] ?= None)
```

Comment:
The `meta-artefact` is internal to how this was calculated - the api would be something like `repoDependencies` - we might need to think about what we are returning when service names and repos don't match (or treat that as out-of-scope for now)